src/main/java/com/pmarko09/medical_clinic_proxy/exception/MedicalClinicServiceUnavailableException.java

```diff
@@ -0,0 +1,7 @@
+package com.pmarko09.medical_clinic_proxy.exception;
+
+public class MedicalClinicServiceUnavailableException extends RuntimeException {
+    public MedicalClinicServiceUnavailableException(String message) {
```

Comment:
jak masz exception który jest obsługiwany w jednym miejscu, to nie lepiej wrzucić message do exception, zamiast przekazywać go w metodzie fallbacka?

Reply:
Mozna, ale wtedy albo masz wszedzie ten sam message do tego wyjatku, albo musisz zrobic dodatkowy konstruktor by 1 byl z domyslnym message, a drugi z customowym. Ja zrobilem tak, bo de facto w roznym fallbacku czy innym miejscu mozesz delikatnie inaczej chciec opisac, co sie dzieje.